email.go

```diff
@@ -36,6 +40,41 @@ Remember to read our guidelines (%s/policy-list) about the requirements your mai
 Thanks for helping us secure email for everyone :)
 `
 
+func makeEmailConfigFromEnv() (emailConfig, error) {
+	// create config
+	c := emailConfig{
+		username:           os.Getenv("SMTP_USERNAME"),
+		password:           os.Getenv("SMTP_PASSWORD"),
+		submissionHostname: os.Getenv("SMTP_ENDPOINT"),
+		port:               os.Getenv("SMTP_PORT"),
+		sender:             os.Getenv("SMTP_FROM_ADDRESS"),
+		website:            os.Getenv("FRONTEND_WEBSITE_LINK"),
```

Comment:
it doesn't seem like we validate `sender` or `website` are set; are they optional?

Reply:
Not really. I'll set defaults for them if unspecified.